### PR TITLE
Removed default ingress annotation

### LINKED ISSUE #54: Remove the default ingress-class
By default the ingress class annotation is set:

```
ingress:
  enabled: false
  annotations:
    # Adjust the ingress class when not using nginx as incress controller
    kubernetes.io/ingress.class: nginx
    # kubernetes.io/tls-acme: "true"
```

This causes issues if you want to unset this value to use the default ingress class.

The only solution I could find was setting annotations to be equal to an empty string, because otherwise helm merges any object you pass in.

### PR DESCRIPTION
**Pull request by bravecobra**
This PR removes the default nginx annotation from the `values.yaml`

Closes #54

### CODE REVIEW

File: charts/localstack/values.yaml [hunk 1]
```diff
@@ -97,9 +97,9 @@ service:
 
 ingress:
   enabled: false
-  annotations:
+  annotations: {}
     # Adjust the ingress class when not using nginx as incress controller
```
Comment (LaikaN57) [inline charts/localstack/values.yaml:101]: Can you docfix this typo while you are in here too please. Thanks.

**Review by LaikaN57 — APPROVED** [review]
LGTM. We may want to add some "extra annotations" like values in the future but we can do that in a follow up PR. If anyone has a suggestion or example of a good pattern for this, please LMK.

**Comment by bravecobra** [comment]
We can add additional annotations by default indeed but they should be localstack specific and never assume any implementation outside the helmchart like which ingresscontroller is to be used. The only way to "remove" this annotation was to overwrite it. The default ingressClass however should have kicked in.
Default annotations could be specifying that each component is part of this helm chart, a practice that is rather common in other charts.

**Comment by whummer** [comment]
Thanks @bravecobra for the fix, and @LaikaN57 for the review! 👍

**Comment by whummer** [comment]
Forgot to mention - we'll need to bump the version to release a new version of the chart. Can either of you please help with that and create a small follow-up PR @bravecobra @LaikaN57 ? Please also add a short note to the changelog. Thanks!

**Pull request merged**